diesel/src/lib.rs

```diff
@@ -196,7 +196,8 @@ pub use prelude::*;
 #[doc(inline)]
 pub use query_builder::debug_query;
 #[doc(inline)]
-pub use query_builder::functions::{delete, insert_into, replace_into, select, sql_query, update};
+pub use query_builder::functions::{delete, insert_into, insert_or_ignore_into, replace_into,
+                                   select, sql_query, update};
```

Comment:
Oh wow is that was rustfmt suggests? (It usually doesn't use visual indentation anywhere)